### PR TITLE
Fix extpos

### PR DESCRIPTION
**Pull request by AngledLuffa**
@nschneid ?

A couple more fixes for ExtPos in PUD

### CODE REVIEW

File: en_pud-ud-test.conllu [hunk 1]
```diff
@@ -14245,9 +14245,9 @@
 2	Macedonian	Macedonian	ADJ	JJ	Degree=Pos	3	amod	3:amod	Proper=True
 3	army	army	NOUN	NN	Number=Sing	4	nsubj	4:nsubj	_
 4	advanced	advance	VERB	VBD	Mood=Ind|Tense=Past|VerbForm=Fin	0	root	0:root	_
-5	as	as	ADV	RB	_	9	case	9:case	_
-6	far	far	ADV	RB	Degree=Pos	5	fixed	5:fixed	_
-7	as	as	ADP	IN	_	5	fixed	5:fixed	_
+5	as	as	ADV	RB	_	6	advmod	6:advmod	_
+6	far	far	ADV	RB	Degree=Pos	4	advmod	4:advmod	_
+7	as	as	SCONJ	IN	_	9	mark	9:mark	_
```
Comment (nschneid) [inline en_pud-ud-test.conllu:14250]: ADP/case I think

File: en_pud-ud-test.conllu [hunk 2]
```diff
@@ -14245,9 +14245,9 @@
...
 8	the	the	DET	DT	Definite=Def|PronType=Art	9	det	9:det	_
 9	pass	pass	NOUN	NN	Number=Sing	4	obl	4:obl:as_far_as|13:nsubj	_
```
Comment (nschneid) [inline en_pud-ud-test.conllu:14252]: attaches to "far", `6:obl:as`

File: en_pud-ud-test.conllu [hunk 3]
```diff
@@ -11754,7 +11754,7 @@
 28	of	of	ADP	IN	_	29	case	29:case	Proper=True
 29	Panama	Panama	PROPN	NNP	Number=Sing	27	nmod	27:nmod:of	SpaceAfter=No
 30	)	)	PUNCT	-RRB-	_	20	punct	20:punct	_
-31	as	as	SCONJ	IN	_	37	mark	37:mark	_
+31	as	as	SCONJ	IN	ExtPos=SCONJ	37	mark	37:mark	_
```
Comment (nschneid) [inline en_pud-ud-test.conllu:11757]: In this context, if "as opposed to" is a fixed expression, it functions like a preposition ("as+opposed+to the North Sea"), so ExtPos=ADP, 37:case, and "Sea" should be 11:obl:as_opposed_to not advcl.

BUT I'm not sure this is actually the grammaticized usage of "as opposed to". The sentence:

- From the 16th century, Europeans visiting the Caribbean region identified the "South Sea" (the Pacific Ocean, to the south of the isthmus of Panama) as opposed to the "North Sea" (the Caribbean Sea, to the north of the same isthmus).

Does this mean 'they identified the South Sea—they did not identify the North Sea'? Or is it meant as a definitional contrast: 'they singled out the South Sea in opposition to the North Sea'? I think the latter is more plausible—see [here](https://en.wikipedia.org/wiki/North_Sea_(disambiguation)):

> from the 16th century, Europeans visiting the region distinguished the "South Sea" (the Pacific Ocean south of the isthmus of Panama) from the "North Sea" (the Caribbean Sea north of the same isthmus).

The latter reading suggests not fixed but rather xcomp(identified, opposed), obl(opposed, North Sea) (cf. "Louise struck me as a fool" [here](https://universaldependencies.org/u/dep/xcomp.html), though a closer analogy would be "Louise struck me as consumed with rage").

File: en_pud-ud-test.conllu [hunk 4]
```diff
@@ -16431,7 +16431,7 @@
 10	first	first	ADJ	JJ	Degree=Pos|NumForm=Word|NumType=Ord	12	amod	12:amod	_
 11	non-white	non-white	ADJ	JJ	Degree=Pos	12	amod	12:amod	_
 12	princess	princess	NOUN	NN	Number=Sing	7	xcomp	7:xcomp	_
-13	as	as	SCONJ	IN	_	19	mark	19:mark	_
+13	as	as	SCONJ	IN	ExtPos=SCONJ	19	mark	19:mark	_
```
Comment (nschneid) [inline en_pud-ud-test.conllu:16434]: Re: the UPOS of "as" in the fixed expression: UniversalDependencies/UD_English-EWT#434

It's unresolved, so fine to stick with SCONJ for now.

**Review by nschneid — CHANGES_REQUESTED** [review]

File: en_pud-ud-test.conllu (hunk 2 above)
Comment (AngledLuffa) [reply en_pud-ud-test.conllu:14252]: Great, thanks.  Keep `13:nsubj` as an enhanced dependency?

File: en_pud-ud-test.conllu (hunk 3 above)
Comment (AngledLuffa) [reply en_pud-ud-test.conllu:11757]: agreed for sure on the 2nd reading

File: en_pud-ud-test.conllu (hunk 2 above)
Comment (nschneid) [reply en_pud-ud-test.conllu:14252]: Yes

File: en_pud-ud-test.conllu (hunk 1 above)
Comment (nschneid) [reply en_pud-ud-test.conllu:14250]: ADP

File: en_pud-ud-test.conllu (hunk 3 above)
Comment (AngledLuffa) [reply en_pud-ud-test.conllu:11757]: so then the `as` would be `case(identified, as)`?  `to` would be `mark(North Sea, to)`?

Comment (AngledLuffa) [reply en_pud-ud-test.conllu:11757]: pushed a change which might cover this (or might be horribly wrong).  is it standard to leave comments on the sentences discussing annotation corner cases?  this seems like a candidate for leaving one regarding `as opposed to` not being fixed

Comment (nschneid) [reply en_pud-ud-test.conllu:11757]: > so then the `as` would be `case(identified, as)`? `to` would be `mark(North Sea, to)`?

The other way around: `mark` is for SCONJ, `case` for ADP. I'd say this "as" is SCONJ.

> pushed a change which might cover this (or might be horribly wrong). is it standard to leave comments on the sentences discussing annotation corner cases? this seems like a candidate for leaving one regarding `as opposed to` not being fixed

Not standard but I don't see any harm if you want to add one

File: en_pud-ud-test.conllu (hunk 2 above)
Comment (nschneid) [reply en_pud-ud-test.conllu:14252]: head should be 6 not 7

File: en_pud-ud-test.conllu [hunk 5]
```diff
@@ -11754,13 +11754,13 @@
...
-32	opposed	oppose	VERB	VBN	Tense=Past|VerbForm=Part	31	fixed	31:fixed	_
-33	to	to	ADP	IN	_	31	fixed	31:fixed	_
+31	as	as	ADP	IN	_	11	case	11:case	_
+32	opposed	oppose	VERB	VBN	Tense=Past|VerbForm=Part	11	xcomp	11:xcomp	_
+33	to	to	ADP	IN	_	37	mark	37:mark	_
 34	the	the	DET	DT	Definite=Def|PronType=Art	37	det	37:det	_
 35	"	"	PUNCT	``	_	37	punct	37:punct	SpaceAfter=No
 36	North	North	PROPN	NNP	Number=Sing	37	compound	37:compound	_
-37	Sea	Sea	PROPN	NNP	Number=Sing	11	advcl	11:advcl:as_opposed_to	SpaceAfter=No
+37	Sea	Sea	PROPN	NNP	Number=Sing	32	obl	32:obl	SpaceAfter=No
```
Comment (nschneid) [inline en_pud-ud-test.conllu:11763]: `32:obl:to`

Comment (AngledLuffa) [reply en_pud-ud-test.conllu:11763]: oops, sorry

File: en_pud-ud-test.conllu (hunk 2 above)
Comment (AngledLuffa) [reply en_pud-ud-test.conllu:14252]: sorry again :/

File: en_pud-ud-test.conllu (hunk 3 above)
Comment (AngledLuffa) [reply en_pud-ud-test.conllu:11757]: Does the current change look better?

File: en_pud-ud-test.conllu [hunk 6]
```diff
@@ -11754,13 +11755,13 @@
 28	of	of	ADP	IN	_	29	case	29:case	Proper=True
 29	Panama	Panama	PROPN	NNP	Number=Sing	27	nmod	27:nmod:of	SpaceAfter=No
 30	)	)	PUNCT	-RRB-	_	20	punct	20:punct	_
-31	as	as	SCONJ	IN	_	37	mark	37:mark	_
-32	opposed	oppose	VERB	VBN	Tense=Past|VerbForm=Part	31	fixed	31:fixed	_
-33	to	to	ADP	IN	_	31	fixed	31:fixed	_
+31	as	as	SCONJ	IN	_	11	mark	11:mark	_
```
Comment (nschneid) [inline en_pud-ud-test.conllu:11758]: "as" marks "opposed", which should be its head

**Review by nschneid — APPROVED** [review]
